truss-chains/tests/test_framework.py

```diff
@@ -99,7 +100,7 @@ class ModelWithRunRemote(chains.ModelBase):
         def run_remote(self) -> str:
             return self.__class__.name
 
-    match = "Models must have a `predict` method."
+    match = "MODELs must have a `predict` method."
```

Comment:
Hm this is a result of the `StrEnum` right? This looks a little weird to me, I'd probably prefer lowercasing or additional processing to make it `Models/Chainlets` in error messages again